src/test/resources/mapping/expectedTransformationFields.json

```diff
@@ -1,1163 +1,1147 @@
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.staffonly.action.note",
-       "displayNameKey": "item.itemNoteTypeId.staffOnly",
-       "referenceDataValue": "Action note",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='0e40884c-3523-4c6d-8187-d578e3d2794e' && ((@.staffOnly) && @.staffOnly == true))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.staffonly.binding",
-       "displayNameKey": "item.itemNoteTypeId.staffOnly",
-       "referenceDataValue": "Binding",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='87c450be-2033-41fb-80ba-dd2409883681' && ((@.staffOnly) && @.staffOnly == true))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.materialtypeid.book",
-       "displayNameKey": "item.materialTypeId",
-       "referenceDataValue": "book",
-       "path": "$.items[*].materialTypeId",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.materialtypeid.dvd",
-       "displayNameKey": "item.materialTypeId",
-       "referenceDataValue": "dvd",
-       "path": "$.items[*].materialTypeId",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.metadata.createdbyuserid",
-       "displayNameKey": "item.metadata.createdByUserId",
-       "path": "$.item.metadata.createdByUserId",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.metadata.createddate",
-       "displayNameKey": "item.metadata.createdDate",
-       "path": "$.item.metadata.createdDate",
-       "recordType": "ITEM",
-       "metadataParameters": {
-         "datesOfPublication": "$.instance.publication[*].dateOfPublication",
-         "languages": "$.instance.languages"
-       }
-     },
-     {
-       "fieldId": "item.metadata.updatedbyuserid",
-       "displayNameKey": "item.metadata.updatedByUserId",
-       "path": "$.item.metadata.updatedByUserId",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.metadata.updateddate",
-       "displayNameKey": "item.metadata.updatedDate",
-       "path": "$.item.metadata.updatedDate",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.numberofpieces",
-       "displayNameKey": "item.numberOfPieces",
-       "path": "$.items[*].numberOfPieces",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.permanentloantypeid.can.circulate",
-       "displayNameKey": "item.permanentLoanTypeId",
-       "referenceDataValue": "Can circulate",
-       "path": "$.items[*].permanentLoanTypeId",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.permanentloantypeid.reading.room",
-       "displayNameKey": "item.permanentLoanTypeId",
-       "referenceDataValue": "Reading room",
-       "path": "$.items[*].permanentLoanTypeId",
-       "recordType": "ITEM"
-     },
```

Comment:
The removed entities for these contain the `referenceDataValue` and the new one does not.

UI pattern for it now looks like `"Item - Permanent loan type - {value}",` where value is `referenceDataValue`. Should the UI be accommodated as well or the `referenceDataValue` is missed by accident?